### DISCUSSION TITLE
What the difference between `DropdownMenuBody` and `DropdownContainer`?

### DISCUSSION THREAD

**Discussion by vasilii-kovalev** [epam/UUI#1115] [General]
According to the documentation, they both are used to wrap the dropdown body content. They both Flexboxes with `flex-direction: column` and white background. The most noticeable discrepancy is different `box-shadow` value.

What component in which situations should be used? For now, it seems `DropdownMenuBody` is more suitable for dropdown body in all the cases.

**Comment by AlekseyManetov** [comment]
`DropdownMenuBody` is used for wrapping directly [DropdownMenu](https://uui.epam.com/documents?id=dropdownMenu&mode=doc&skin=UUI4_promo&category=components) component items, it's provide additional handlers for keyboard navigation between items and handle focus/blur. On the other side, `DropdownContainer` it's just a wrapper for common dropdowns, which provides just some style configuration for you, that you won't create your own css for every such case.
